src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/TypeMetadataNode.cs

```diff
@@ -48,7 +48,9 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
             {
                 // A delegate type metadata is rather useless without the Invoke method.
                 // If someone reflects on a delegate, chances are they're going to look at the signature.
-                dependencies.Add(factory.MethodMetadata(_type.GetMethod("Invoke", null)), "Delegate invoke method metadata");
+                var methodDesc = _type.GetMethod("Invoke", null);
```

Comment:
This doesn't look like part of this change

Reply:
reverted.